src/components/RoomCard.tsx

```diff
@@ -53,22 +59,32 @@ export const RoomCard: React.FC<{
   const numberOfDays = 1;
   const roomsNumber = 1;
 
-  const handleBook = () => {
-    const id = uuidv4();
-    dispatch({
-      type: 'SET_CHECKOUT',
-      payload: {
-        id,
-        spaceId: roomId,
-        facilityId,
-        from: 'today',
-        to: 'tomorrow',
-        roomsNumber: 2,
-        timestamp: 0
-      }
-    });
-    navigate(`/checkout/${id}`);
-  };
+  const [loading, setLoading] = useState<boolean>(false);
+  const [error, setError] = useState<undefined | string>();
+
+  const handleBook = useCallback(async () => {
+    try {
+      setError(undefined);
+      setLoading(true);
+
+      const res = await axios.post(backend.url + '/offers/' + offer.id + '/price');
+      dispatch({
+        type: 'SET_CHECKOUT',
+        payload: {
+          facilityId,
+          pricedOffer: res.data
+        }
+      });
+
+      logger.info('Get priced offer successfully');
```

Comment:
when do we use the logger?

And are these logged stored somewhere on a server?

Reply:
This is console logger

Reply:
This logs for development only. Not saving